testsuite/integration/web/src/test/config/arq/arquillian-bootable.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<arquillian xmlns="http://jboss.org/schema/arquillian" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+    <defaultProtocol type="jmx-as7" />
+
+    <container qualifier="jboss" default="true">
+        <configuration>
+            <property name="installDir">${install.dir}</property>
+            <property name="jarFile">${bootable.jar}</property>
+
+            <property name="javaVmArguments">${server.jvm.args}</property>
```

Comment:
is this line compliant with configuration settings defined here? https://github.com/wildfly/wildfly/blob/master/testsuite/integration/web/src/test/config/arq/arquillian.xml#L10

Reply:
Thanks, line replaced.

Reply:
It is still there, as well :)

Reply:
Fixed in next commit! Sorry, added only pom into the commit...